packages/core/src/lib/select/select-input.svelte

```diff
@@ -58,18 +85,24 @@ $: errorClasses =
     on:input
     on:keydown
     on:focus
+    on:blur
     on:mousemove
   />
   <button
-    class="absolute right-2 top-1.5"
+    type="button"
+    class="absolute right-2 top-1/2 -translate-y-1/2 transform"
     tabindex="-1"
     aria-label="Toggle menu"
+    aria-controls={menuId}
+    aria-expanded={isOpen}
     on:click
     on:keydown
+    on:mousedown|preventDefault
+    on:pointerdown|preventDefault
```

Comment:
These prevent focus from happening, so focus remains in the combobox even as the menu is toggled open and closed. `click` events continue to fire